From 02892690e6866781a0a6c84e25a2c1d277822788 Mon Sep 17 00:00:00 2001
From: Ben Pfaff <blp@nicira.com>
Date: Wed, 22 Sep 2010 13:16:07 -0700
Subject: [PATCH 6/6] xenserver: Now everything is free.

Signed-off-by: Ben Pfaff <blp@nicira.com>
---
 COPYING          |    3 +--
 xenserver/README |    7 ++-----
 2 files changed, 3 insertions(+), 7 deletions(-)

diff --git a/COPYING b/COPYING
index 375efec..8d1bc9d 100644
--- a/COPYING
+++ b/COPYING
@@ -20,8 +20,7 @@ Files under the datapath directory are licensed under the GNU General
 Public License, version 2.
 
 Files under the xenserver directory are licensed on a file-by-file
-basis.  Some files are under an uncertain license that may not be
-DFSG-compliant or GPL-compatible.  Refer to each file for details.
+basis.  Refer to each file for details.
 
 Files lib/sflow*.[ch] are licensed under the terms of the InMon sFlow
 licence that is available at:
diff --git a/xenserver/README b/xenserver/README
index 9fcdb1a..941c4e1 100644
--- a/xenserver/README
+++ b/xenserver/README
@@ -1,11 +1,8 @@
 This directory contains files for seamless integration of Open vSwitch on
 Citrix XenServer hosts managed by the Citrix management tools.
 
-Some of these files are modifications of Citrix's proprietary code.
-Citrix has given permission to distribute these modified files.
-Citrix has not specified a particular license for them.  There is no
-guarantee that, should Citrix specify a license, that it would be
-DFSG-compliant or GPL-compatible.
+Files in this directory are licensed on a file-by-file basis.  Please
+refer to each file for details.
 
 Most of the files in this directory is installed on a XenServer system
 under the same name, if underscores are replaced by slashes.  The
-- 
1.7.1

