--- a/include/openvswitch/ofp-util.h
+++ b/include/openvswitch/ofp-util.h
@@ -189,7 +189,8 @@
 #define OFPUTIL_SUPPORTED_VERSIONS ((1u << OFP10_VERSION) | \
                                     (1u << OFP11_VERSION) | \
                                     (1u << OFP12_VERSION) | \
-                                    (1u << OFP13_VERSION))
+                                    (1u << OFP13_VERSION) | \
+                                    (1u << OFP14_VERSION))
 #define OFPUTIL_DEFAULT_VERSIONS OFPUTIL_SUPPORTED_VERSIONS
 
 enum ofputil_protocol ofputil_protocols_from_string(const char *s);
